docs/en/sql-reference/table-functions/format.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: /en/sql-reference/table-functions/format
+sidebar_position: 56
+sidebar_label: format
+---
+
+# format
+
+Extracts table structure from data and parses it according to specified input format.
+
+**Syntax**
+
+``` sql
+format(format_name, data)
+```
+
+**Parameters**
+
+-   `format_name` — The [format](../../interfaces/formats.md#formats) of the data.
+-   `data` — String literal containing data in specified format
```

Comment:
Do we support column as well as input?

Reply:
Table functions cannot get columns from other tables.
Maybe you meant constant expressions?

Looks like yes:

```
milovidov-desktop :) SELECT * FROM format(LineAsString, 'Hello' || 'World')

┌─line───────┐
│ HelloWorld │
└────────────┘

1 row in set. Elapsed: 0.001 sec.
```